src/_cffi_src/openssl/ssl.py

```diff
@@ -468,6 +476,13 @@
 long DTLSv1_handle_timeout(SSL *);
 long DTLS_set_link_mtu(SSL *, long);
 long DTLS_get_link_min_mtu(SSL *);
+size_t DTLS_get_data_mtu(SSL *);
+long SSL_set_mtu(SSL *, long);
+typedef ... BIO_ADDR;
```

Comment:
This should be in types

Reply:
Where are these BIO_ADDR things originally defined? If bio.h they should live in bio.py

Reply:
Fixed